cbor.js

```diff
@@ -389,8 +390,8 @@ function decode(data, tagger, simpleValue) {
   }
 
   var ret = decodeItem();
-  if (offset !== data.byteLength)
-    throw "Remaining bytes";
+  if (offset !== data.byteLength && !allowRemainingBytes)
+    throw (data.byteLength - offset) + " remaining bytes after end of encoding";
```

Comment:
if remaining bytes are allowed, just return the item
if not allowed, report how many bytes remain